### ISSUE TITLE
Make text on comment form clearer about the comment also going to Council. Copy in comments

### ISSUE THREAD

**Issue by JoannaHill** [openaustralia/planningalerts#1841]

**Comment by katska** [comment]
@JoannaHill ?

**Comment by JoannaHill** [comment]
@katska this was originally your comment I've moved over from Bugherd. Is it still relevant?

**Comment by katska** [comment]
@JoannaHill Yes if this was not discussed before going live. 
If an active decision was made, I'd like to understand the thinking :)

**Comment by JoannaHill** [comment]
I don't think it was particularly active decision, probably a rushed one to get it out the door at the time. I'm very happy with your copy suggestion. So we're going with this below?

"Your comment will be published here and automatically sent to Byron Shire Council. They may consider your comments along with other submissions when making any decision on this application."

**Comment by JoannaHill** [comment]
Text was made live some time ago. Closing this issue.
